### ISSUE TITLE
WKBElement class has no __bool__ method

### ISSUE THREAD

**Issue by alkment** [geoalchemy/geoalchemy2#243]
It seems like currently WKBElment objects returned by queries don't have a __bool__ method thus it's not easy to implement logic that conditions on the ORM model having a non null GIS field.

What do you think about adding the following ?

```
def __bool__(self):
    return True
```

**Comment by adrien-berchet** [comment]
I am not sure of what you want to achieve but can't you just check whether your wkb object is None or not?

**Comment by alkment** [comment]
Checking for None is what I am currently doing.
Other elements returned by sqlalchemy generally have a `__bool__` method.   So I thought it would be nice if wkb has one too.
If you think there is a reason why it should not close this.

**Comment by adrien-berchet** [comment]
Ok, I see your point.
It could be implemented but we have to think a little more about what a WKBElement returning `True` really is. For example, should a WKBElement containing an empty Geometry still return `True`? Usually the empty objects return `False`.

**Comment by alkment** [comment]
Yes simply returning `True` might not make sense especially if the Geometry can be empty.
There could be other instances when returning `False` might make sense. I am not an expert on this datatype.
